react-components/package.json

```diff
@@ -43,7 +43,8 @@
   },
   "scripts": {
     "start": "npm run storybook",
-    "build": "node scripts/build.js",
+    "build": "REACT_APP_MODE=development node scripts/build.js",
+    "prod": "REACT_APP_MODE=production node scripts/build.js",
```

Comment:
The changes to this file shouldn't be necessary, since the `development` env is used by CRA when running `npm start` and launching the react app in a local dev session. We don't need that because we use `storybook` for local dev instead.

If you create a React app and eject it, you'll see a `build.js` file (like the one in this PR) and a `start.js` script. That `start.js` script is where CRA sets `process.env.BABEL_ENV = 'development'` and `process.env.NODE_ENV = 'development'`.

They used to have separate `webpack.config.dev.js` and `webpack.config.prod.js` files for dev vs. prod, and that's why we are just using one named `webpack.config.prod.js` (from when we ejected a CRA v1.0.14 app). In CRA 2.x, it looks like they've combined these files into one, and are using those `isEnvX` vars to tweak the configs. We only need the prod configs, though our time is probably best spent on something other than refactoring all those config changes for a prod-only config 😅